lib/app/store/user.js

```diff
@@ -1,8 +1,32 @@
+const got = require('got')
 const INIT = { credentials: 'include', headers: { accept: 'application/json' } }
 
 module.exports = function user () {
   return (state, emitter) => {
     state.user = state.user || {}
+    state.user.notificationsAmount = -1
+
+    var rerender = () => emitter.emit('render')
+
+    if (state.user.isAuthenticated) {
+      let requestOptions = {retry: 0,
+        headers: {brfauth: state.user.forumAuthenticationToken, 'Cache-Control': 'no-store'}}
+
+      got(process.env.FORUM_URL + '/api/unread', requestOptions)
+      .then((resp) => {
+        var json = JSON.parse(resp.body)
+        console.log(json)
```

Comment:
We should really try and avoid excessive logging, especially in production code.